src/components/IconTextIcon/IconTextIcon.scss

```diff
@@ -1,11 +1,6 @@
+@use "../../tokens/spacings/spacings" as *;
+
 .moonstone-IconTextIcon {
```

Comment:
I am wondering if we should use we should use a `$spacing-icon_big` spacing/gap when we set `iconSize` to `big`

Reply:
When icons of `iconTextIcon` are set to size big there is no extra className added to the component, since the spacing is directly assigned as a `gap` on `.moonstone-IconTextIcon` I don't have an efficient way of changing it.